src/style.css

```diff
@@ -1,3 +1,108 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 0;
+  text-align: center;
+}
+
+.conteiner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+  text-align: center;
+}
+
+a {
+  display: inline-block;
+  font-size: 12px;
+}
+
+a.img {
+  height: 60px;
+  width: 40px;
+}
+
+img {
+  height: 40px;
+  width: 40px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+
+  position: relative;
+  left: 0%;
+
+  text-decoration: none;
+}
+
+.class_item {
+  display: inline;
+}
+
+.nav_link {
+  text-align: center;
+  height: 60px;
+
+  margin-left: 10px;
+  margin-right: 10px;
+  color: black;
+  text-decoration: none;
+
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+}
+
+a.hover:hover {
```

Comment:
Consider using class selector here to add hover style for all links in the navigation